### PR TITLE
Make toggle

### PR DESCRIPTION
**Pull request by jonleesy**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -33,6 +34,13 @@
 
   <!-- dropdown menu for election year selection -->
   <select id="dropdown-year"></select>
+  
+
+  <!-- Rounded switch -->
+  <div class="toggle">
```
Comment (kymckay) [inline public/index.html]: Think checkbox should go beside source dropdown

**Comment by kymckay** [comment]
@jonleesy I fixed the code, but the start state is currently inverted (it starts checked, but should be unchecked as "Actual")

**Comment by jonleesy** [comment]
@kymckay should the toggle be "beautified" in this branched or should we just merged it first and do it as a feature?

**Comment by kymckay** [comment]
@jonleesy Probably merge first, but I think it should be moved to beside the sources dropdown

**Comment by jonleesy** [comment]
I'm actually working on it already. And I've managed to have it on the map

**Comment by jonleesy** [comment]
@kymckay would be great if you could review, then I could take screenshots from mains and draft the document

**Review by kymckay — APPROVED** [review]
Nice, looks decent.

I see what you mean about interacting with the map though, it does seem to be something on this branch causing it 🤔

File: public/index.html (hunk 1 above)
Comment (jonleesy) [reply public/index.html]: Hahah. Guess we're not doing it on the map anymore?

**Comment by jonleesy** [comment]
> Nice, looks decent.
> 
> I see what you mean about interacting with the map though, it does seem to be something on this branch causing it 🤔

I think I had a function that keeps constituencies highlighted upon click

**Comment by kymckay** [comment]
>I think I had a function that keeps constituencies highlighted upon click

Nah it seems like the cursor is interactive a fixed distance above where it actually is (like constituencies become highlighted if you hover below them)

**Comment by kymckay** [comment]
I don't see anything in the code, I think it must be something about the HTML structure that mapbox doesn't like

**Comment by kymckay** [comment]
Yeah confirmed, if you load the page and inspect the toggle and delete it suddenly the pointer starts working again. I think you need to put the toggle element outside of the `id="map"` div maybe

**Comment by jonleesy** [comment]
image

tried it, it still does it

**Comment by jonleesy** [comment]
Ill take that back. Somehow its better.

**Comment by jonleesy** [comment]
@kymckay Its done. Place it on top instead. Not sure how the dropdown is configured, seemed pretty stubborn. I think I have given up

**Comment by jonleesy** [comment]
image
Looks like this now

**Comment by kymckay** [comment]
@jonleesy Nice, I'll attempt to tweak the placement a bit then merge

**Comment by kymckay** [comment]
You need to push changes though

**Comment by kymckay** [comment]
I couldn't be bothered trying to properly lay the toggle out so just made the labelling more clear that it's for the map to improve affordance 😅

**Pull request merged**